docs/relnotes.md

```diff
@@ -7,18 +7,66 @@ hide:
+## Community edition: 2.0
+
+Published on TO_BE_DEFINED.
+
+[`taipy` 2.0](https://pypi.org/project/taipy/2.0.0/) contains the latest
+[`taipy-config` 2.0](https://pypi.org/project/taipy-config/2.0.0/),
+[`taipy-gui` 2.0](https://pypi.org/project/taipy-gui/2.0.0/),
+[`taipy-core` 2.0](https://pypi.org/project/taipy-core/2.0.0/) and
+[`taipy-rest` 2.0](https://pypi.org/project/taipy-rest/2.0.0/) packages.
+
+#### Features
+
+**`taipy-core`** 2.0.0
+
+   - New data node: SQLTableDataNode. It represents a table in a SQL database.
+   - New data node: JSONDataNode. It represents the data from a JSON file.
+   - SQLDataNode behavior is changed due to the release of SQLTableDataNode. Now it represents the data
+     using custom read and write queries.
+   - In standalone mode, a job whose status is `SUBMITTED`, `PENDING`, or `BLOCKED`, can be canceled. When
+     canceling the job, its subsequent jobs will be abandoned and their statuses will be set to `ABANDONED`. When
+     the cancel method is called on a job with its status being either `RUNNING`, `COMPLETED`, or `SKIPPED`, its
+     subsequent jobs will be abandoned while its status remains unchanged. A job whose status is `FAILED`,
+     `CANCELED`, or `ABANDONED` cannot be canceled.
+   - Taipy Core can now be run as a service by using `Core().run()` or `tp.run(Core())`. By running Core as a service, Taipy will initialize the scheduler and the job dispatcher based on the config provided. Taipy Core can also be run along with Taipy Gui or Taipy Rest.
 
 #### Improvements
 
-**`taipy-core`**<br/>1.2.0
+**`taipy-core`** 2.0.0
 
-   - The data node of a scenario or a pipeline can now be accessed at the scenario or pipeline levels.
+   - The data node of a scenario or a pipeline can now be accessed directly at the scenario or pipeline levels.
    - When submitting a scenario, a pipeline or a task, a list of jobs or a Job will be returned.
-   - In standalone mode, a job whose status is `SUBMITTED`, `PENDING`, or `BLOCKED`, can be canceled. When canceling the job, its subsequent jobs will be abandoned. On the other hand, a job whose status is `RUNNING`, `COMPLETED`, `SKIPPED`, `FAILED`, `CANCELED`, or `ABANDONED`, cannot be canceled. When the cancel method is called on a job with its status being either `RUNNING`, `COMPLETED`, or `SKIPPED`, its subsequent jobs will be abandoned while its status remains unchanged.
-   - When submitting a scenario, pipeline, or task in standalone mode, the user can use the parameters _wait_ and _timeout_ to wait until the submitted jobs are finished or up to _timeout_ seconds.
+   - When submitting a scenario, pipeline, or task in standalone mode, the user can use the parameters _wait_ and
+     _timeout_ to wait until the submitted jobs are finished or up to _timeout_ seconds.
+   - When in standalone mode, the job dispatcher will be running in a sub-thread that will periodically check for new jobs submitted by Taipy to execute.
```

Comment:
'the job dispatcher will be running in a sub-thread that will periodically check'
  -> 
  'the job dispatcher runs in a sub-thread that periodically checks'